src/main/java/com/session/service/Main.java

```diff
@@ -47,4 +67,23 @@ public static void main(String[] args) {
         }
     }
 
+    private static void startApp() throws Exception {
+        initLogging();
+
+        info().log("starting sessions-service-client");
+    }
+
+    private static void initLogging() throws LoggingException {
+        LogSerialiser serialiser = new JacksonLogSerialiser();
+        LogStore store = new MDCLogStore(serialiser);
+        Logger logger = new LoggerImpl("sessions-service-client");
+
+        DPLogger.init(new Builder()
+                .logger(logger)
+                .logStore(store)
+                .serialiser(serialiser)
+                .dataNamespace("sessions-service-client.data")
+                .create());
+    }
+
```

Comment:
This wont be needed in the client as the app using this library should be doing this already. Fine to leave here for testing though.